crates/testenv/src/lib.rs

```diff
@@ -11,6 +11,7 @@ use bitcoincore_rpc::{
     bitcoincore_rpc_json::{GetBlockTemplateModes, GetBlockTemplateRules},
     RpcApi,
 };
+pub use electrsd;
```

Comment:
Since we are exporting for convenience, could we also export all the useful "nested" crates as well?

```suggestion
pub use electrsd;
pub use electrsd::bitcoind;
pub use electrsd::bitcoind::anyhow;
pub use electrsd::bitcoind::bitcoincore_rpc;
pub use electrsd::electrum_client;
```

Then we can use these for the rest of the codebase.